mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java

```diff
@@ -35,28 +39,48 @@
     private String mPassword;
     private static MyAsyncHttpClient mAsyncHttpClient;
 
+
+
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        Log.d(TAG, "onCreate()");
-        Util.setActivityTheme(this);
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.openhabinfo);
-        mAsyncHttpClient = new MyAsyncHttpClient(this);
-        mOpenHABVersionText = (TextView)findViewById(R.id.openhab_version);
-        mOpenHABUUIDText = (TextView)findViewById(R.id.openhab_uuid);
-        mOpenHABSecretText = (TextView)findViewById(R.id.openhab_secret);
-        mOpenHABSecretLabel = (TextView)findViewById(R.id.openhab_secret_label);
-        if (getIntent().hasExtra("openHABBaseUrl")) {
-            mOpenHABBaseUrl = getIntent().getStringExtra("openHABBaseUrl");
-            mUsername = getIntent().getStringExtra("username");
-            mPassword = getIntent().getStringExtra("password");
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.openhabinfo, container);
+        mAsyncHttpClient = new MyAsyncHttpClient(getActivity().getApplicationContext());
+        mOpenHABVersionText = (TextView)view.findViewById(R.id.openhab_version);
+        mOpenHABUUIDText = (TextView)view.findViewById(R.id.openhab_uuid);
+        mOpenHABSecretText = (TextView)view.findViewById(R.id.openhab_secret);
+        mOpenHABSecretLabel = (TextView)view.findViewById(R.id.openhab_secret_label);
+        Bundle bundle=getArguments();
+
+        if (bundle!=null)
+        {
```

Comment:
I know this is a bit of a nick-pick, but can you format you code to match the current style, with the '{' being on the same line as the statement?